Library/Homebrew/dev-cmd/dispatch-build-bottle.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require "cli/parser"
+require "utils/github"
+
+module Homebrew
+  module_function
+
+  def macos_versions
+    ["10.13", "10.14", "10.15"]
```

Comment:
Could we avoid hardcoding these and instead just allow the job to fail if there isn't a worker? Would be nice to match the format to ensure it's correct, though.

Reply:
Thought about that. Yep, could delete this altogether.

Reply:
Yeh, do that for now I reckon.